tests/io/test_cif.py

```diff
@@ -998,3 +998,68 @@ def test_bibtex(self):
 }
 """
         assert self.mcif_ncl.get_bibtex_string() == ref_bibtex_string
+
+
+class TestCifParserAssess(PymatgenTest):
```

Comment:
I think we can merge this into the `TestCifParser` class given `CifParserAssess` is no longer a standalone class.

Reply:
done